main.py

```diff
@@ -1,17 +1,51 @@
 import animate
 import pygame
 from pygame.locals import *
+from menu import *
+from TicTacToe import *
 
 class App:
     def __init__(self):
+        # Flag to indicate if the whole game/software is running
         self._running = True
+        # Flag to indicate if the game (after you click "start  game") is playing
+        self.playing = False
         self._clock = pygame.time.Clock()
 
         self._display_surf = None
         self.size = self.width, self.height = 640, 400
         self.FPS = 60
 
-        self.avatar = None
```

Comment:
In Python, it is not necessary to declare class attributes, but I think it's nevertheless good Python style. All attributes should be initialized within __init__(self) so that we have an overview.